lib/plugins/inspec-init/lib/inspec-init/templates/profile_gcp/README.md

```diff
@@ -0,0 +1,65 @@
+# Example InSpec Profile For GCP
+
+This example shows the implementation of an InSpec profile for GCP that depends on the [InSpec GCP Resource Pack](https://github.com/inspec/inspec-gcp).  See the [README](https://github.com/inspec/inspec-gcp) for instructions on setting up appropriate GCP credentials.
+
+##  Create a profile 
+
+```
+$ inspec init profile_gcp gcp-profile
```

Comment:
I don't like this CLI interface.  I think it should be:

`inspec init profile --platform gcp my-profile`

I think the `inspec-init` plugin should be modified to accommodate that.

Reply:
I've done so on #3491